### ISSUE TITLE
I can't find the library DOIT Esp32 DevKit v1

### ISSUE THREAD

**Issue by comkin** [espressif/arduino-esp32#4068]
![image](https://user-images.githubusercontent.com/42239154/83953054-9127d680-a83d-11ea-957e-9db3968ad45d.png)
I can't find on arduino ide the library I also added the following links for the board manager
(https://dl.espressif.com/dl/package_esp32_index.json
http://arduino.esp8266.com/stable/package_esp8266com_index.json)

**Comment by lbernstone** [comment]
It is not a library.  It is a board.

**Comment by comkin** [comment]
> It is not a library. It is a board.

yes Yes, but when I insert this library #include <ESP8266WiFi.h> tells me that the library doesn't exist

**Comment by lbernstone** [comment]
ESP32 != ESP8266
